### PR TITLE
Initialize project for first release of SMART ASCVD Risk Calculator

### PR DESCRIPTION
**Pull request by zplata**
This proposal is for initializing the first release of the SMART ASCVD Risk Calculator application. This application is a React web application that simulates a risk score given a variety of vitals and background factors that estimates a percentage of possible heart disease or stroke within the next 10 years and/or the lifetime of a user. 

This proposal includes instructions on how to run the project locally, build the files, and run unit and automated tests, among many other features (specified in the README.md file). 

@kolkheang 
@shriniketsarkar 
@mjhenkes 
@koushic88
@kpshek

### CODE REVIEW

File: app/load_fhir_data.js [hunk 1]
```diff
@@ -0,0 +1,726 @@
...
+    PatientInfo.totalCholesterol = undefined;
+    PatientInfo.hdl = undefined;
+    PatientInfo.systolicBloodPressure = undefined;
+    ASCVDRisk.hideDemoBanner = true;
+    ASCVDRisk.ret.resolve(PatientInfo);
+  };
+  ASCVDRisk.setDefaultPatient = setDefaultPatient;
+
+  /**
+   * Initiate Canadarm logging for the application to output on the console and log specified
```
Comment (shriniketsarkar) [inline app/load_fhir_data.js]: Need to update the comment to may be ->  "Initializer function for Canadarm Logger. 
Initiates the Canadarm logging tool to output logs to the console and the URL specified in the config.js file (in root directory).

File: app/load_fhir_data.js [hunk 2]
```diff
@@ -0,0 +1,726 @@
...
+   * Initiates the Canadarm logging tool to output logs to the console and
+   * the URL specified in the config.js file (root directory).
+   */
+  const initializeLogger = () => {
+    Canadarm.init({
+      onError: false,
```
Comment (shriniketsarkar) [inline app/load_fhir_data.js]: Need to set this to "true" to enable window.onError logs along with Canadarm Logs.

File: app/polyfill.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+      // 7. Return undefined.
+      return undefined;
+    },
+  });
+}
+
+if (typeof Object.assign !== 'function') {
+  Object.assign = function (target, varArgs) {
+ // .length of function is 2
```
Comment (shriniketsarkar) [inline app/polyfill.js]: Do we want to get rid of this comment ?

File: app/load_fhir_data.js (hunk 1 above)
Comment (zplata) [inline app/load_fhir_data.js]: Edited in newest commit

File: app/load_fhir_data.js (hunk 2 above)
Comment (zplata) [inline app/load_fhir_data.js]: Edited in newest commit

File: app/polyfill.js (hunk 3 above)
Comment (zplata) [inline app/polyfill.js]: Deleted in newest commit

**Pull request merged**